gen/generate.jl

```diff
@@ -1,66 +1,35 @@
-using Clang.cindex
-using Clang.wrap_c
-using Printf
+using Pkg
+using Pkg.Artifacts
+using Clang.Generators
+using Clang.Generators.JLLEnvs
+using LibCURL_jll
 
-# Set these to correspond to your local filesystem's curl and clang include paths
-const CURL_PATH = "/usr/local/opt/curl/include/curl"
-const CLANG_INCLUDES = [
-    "/usr/local/opt/llvm/include",
-]
+cd(@__DIR__)
 
-const SRC_DIR = abspath(@__DIR__, "..", "src")
+artifact_toml = joinpath(dirname(Pkg.pathof(LibCURL_jll)), "..", "StdlibArtifacts.toml")
```

Comment:
I may be wrong about this but can't you just use `@__DIR__` here?
```suggestion
artifact_toml = joinpath(@__DIR__, "..", "StdlibArtifacts.toml")
```

Reply:
That's the dir of `LibCURL_jll`, not `LibCURL.jl`. :)